task13/grid.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./grid.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+    <div class="wrap">
+
+        <div class="cart-menu">
+            
+            <div class="cart-menu__img">
+            <img src="./p_O.jpg" alt="">
+            </div>
+            <div class="cart-menu__item title">
```

Comment:
May be will be better create new element? cart-menu__item-title?

Reply:
these are two classes